src/openloco/windows/station_list.cpp

```diff
@@ -526,10 +526,9 @@ namespace openloco::ui::windows::station_list
                 string_ids::label_icons_rail_road_air_water,
             };
 
-            // TODO(avgeffen): I'm not seeing the town name drawn on screen.
             _common_format_args[0] = string_ids::stringid_stringid;
             _common_format_args[1] = station->name;
-            _common_format_args[2] = townmgr::get(station->town)->name;
+            _common_format_args[2] = station->town;
```

Comment:
This change suggests that `station->town` shouldn't be of the type `town_id_t` but rather `string_id`. Agreed?

Reply:
No. You're right in that `string_ids::stringid_stringid;` needs two string ids to be used for the display. The first is the station name, but the second one isn't the town name, it's the icon string. The town ID is used as an argument to the station name.  station.name can be a value like `TOWN_NAMES_START + 281`. Then follow this fragment to see what happens:

<https://github.com/OpenLoco/OpenLoco/blob/ddf564f447fc416262715b0f856011489850d6bc/src/openloco/localisation/stringmgr.cpp#L563-L570>

Reply:
Ah, thanks — I'd completely forgotten about that bit. (And to think, I implemented that…!)